<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>715</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="pbConnect">
       <property name="text">
        <string>连接服务器</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>规约输入框</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>远端IP地址</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pbParse">
       <property name="text">
        <string>解析</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="lePort"/>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="pbSend">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="leIPAddr">
       <property name="minimumSize">
        <size>
         <width>391</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>连接状态    </string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="4">
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QListWidget" name="lwLog">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="layoutMode">
         <enum>QListView::SinglePass</enum>
        </property>
       </widget>
       <widget class="QTableWidget" name="tw">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="lbStatus">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="leText"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>日志</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>端口</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pbCopyLog">
       <property name="text">
        <string>复制日志</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QSplitter" name="splitter_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="pbSendStartDtAct">
        <property name="text">
         <string>U帧启动</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbGeneralInterrogationAct">
        <property name="text">
         <string>总召唤</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbGeneralInterrogationEnd">
        <property name="text">
         <string>总召唤结束</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbSendCommand">
        <property name="text">
         <string>发送命令</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>命令类型标识</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
